.azure-pipelines/scripts/tutorial-checks.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
Can you add some comments into this script explaining what the functions are doing please? It's not immediately clear what it's checking for